### DISCUSSION TITLE
idea chatgpt 插件浏览器的的使用不了怎么解决

### DISCUSSION THREAD

**Discussion by gyk185yyds** [obiscr/ChatGPT#77] [Q&A]
idea chatgpt 插件浏览器的的使用不了怎么解决

**Comment by gyk185yyds** [comment]
![image](https://user-images.githubusercontent.com/124658969/221137046-11063bfa-db8d-4091-8219-f55df556ab62.png)
VPN全局也没用

**Reply by Sisyphean-a** [reply]
老哥，直接暴露ip地址不太好吧，而且你ip地址显示的是郑州啊。最后，点击设置，可以选择代理
